contents/docs/product-analytics/identify/reset.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Reset after logout
```

Comment:
Not sold on this title, but I think it could be useful beyond just after logout. 
```suggestion
title: Resetting persons
```